cicd/gitops/viewing-argo-cd-logs-using-the-kibana-dashboard.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+[id="viewing-argo-cd-logs-using-the-kibana-dashboard"]
+= Viewing Argo CD logs using the Kibana dashboard
+include::_attributes/common-attributes.adoc[]
+:context: viewing-argo-cd-logs-using-the-kibana-dashboard
+
+toc::[]
+
+You can view the Argo CD logs with {logging-title}. The {logging} provides a visualization of logs using Kibana dashboard. To integrate Argo CD with {logging}, the default configuration for the OpenShift Logging Operator enables logging with Argo CD.
+
+[id="prerequisites_viewing-argo-cd-logs-with-the-kibana-dashboard"]
+== Prerequisites
+
+* The {gitops-title} Operator is installed in your cluster.
+* The {logging-title} is installed with default configuration  in your cluster.
+
+include::modules/gitops-storing-and-retrieving-argo-cd-logs.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_viewing-argo-cd-logs-with-the-kibana-dashboard"]
+== Additional resources
+* xref:../../logging/cluster-logging-deploying.adoc#cluster-logging-deploy-console_cluster-logging-deploying[Installing the {logging-title} using the web console]
```

Comment:
You can actually get rid of this completely by incorporating the link inline in the relevant prerequisite item. For example:
```
* The {logging-title} is installed with default configuration in your cluster. See xref:../../logging/cluster-logging-deploying.adoc#cluster-logging-deploy-console_cluster-logging-deploying[Installing the {logging-title} using the web console].
```
or
```
* The {logging-title} is xref:../../logging/cluster-logging-deploying.adoc#cluster-logging-deploy-console_cluster-logging-deploying[installed with default configuration] in your cluster.
```

Reply:
I agree with that approach. However, because we should not use xrefs in modules (as explained here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-procedures), I suggest leaving prerequisites in the assembly to include the xref as you suggested here. Thank you!